Assignment1.py

```diff
@@ -25,6 +29,9 @@
 
 fig5 = px.ecdf(df, x="SepalLengthCm", y="SepalWidthCm", color="Species", ecdfnorm=None)
 
```

Comment:
After all of these do fig1.show(), fig2.show(), etc. The code runs and I know it is right but the plots don't show up when I run the code on mine.